corehq/apps/hqwebapp/views.py

```diff
@@ -1276,3 +1280,49 @@ def temporary_google_verify(request):
     # will remove once google search console verify process completes
     # BMB 4/20/18
     return render(request, "google9633af922b8b0064.html")
+
+
+@require_POST
+@csrf_exempt
+def log_email_event(request):
+    # From Amazon SNS:
+    # https://docs.aws.amazon.com/ses/latest/DeveloperGuide/event-publishing-retrieving-sns-examples.html
+    request_json = json.loads(request.body)
+
+    if request_json['Type'] == "SubscriptionConfirmation":
+        # When creating an SNS topic, the first message is a subscription
+        # confirmation, where we need to access the subscribe URL to confirm we
+        # are able to receive messages at this endpoint
+        subscribe_url = request_json['SubscribeURL']
+        requests.get(subscribe_url)
+        return HttpResponse()
+
+    message = json.loads(request_json['Message'])
+    headers = message.get('mail', {}).get('headers', [])
+
+    subevent_id = None
+    for header in headers:
+        if header["name"] == COMMCARE_MESSAGE_ID_HEADER:
+            subevent_id = header["value"]
+            break
+
+    if subevent_id is None:
```

Comment:
nit
```suggestion
    else:
```

Reply:
I always forget that `for/else` exists, thanks!